### PR TITLE
EPMDJ-5304: added no plugins stub to service group dashboard page

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-5304

### CODE REVIEW

File: src/modules/no-plugins-stub/no-plugins-stub.tsx [hunk 1]
```diff
@@ -3,30 +3,29 @@ import { BEM } from '@redneckz/react-bem-helper';
...
-import { Agent } from 'types/agent';
+import { spacesToDashes } from 'utils';
 
 import styles from './no-plugins-stub.module.scss';
 
 interface Props {
   className?: string;
-  agent: Agent;
+  agentId?: string;
+  agentType: 'Agent' | 'Service group';
```
Comment (pavyarov) [inline src/modules/no-plugins-stub/no-plugins-stub.tsx:13]: Service group everywhere is `ServiceGroup`

File: src/pages/service-group-full-page/dashboard/dashboard.module.scss [hunk 2]
```diff
@@ -1,13 +1,13 @@
 @import '~common/style-guide/index';
 
-.dashboard-page {
+.dashboard-service-group-page {
```
Comment (pavyarov) [inline src/pages/service-group-full-page/dashboard/dashboard.module.scss:3]: `service-group-dashboard-page`

**Review by pavyarov — CHANGES_REQUESTED** [review]
``

**Review by SMPlay — APPROVED** [review]

File: src/modules/no-plugins-stub/no-plugins-stub.tsx [hunk 3]
```diff
@@ -3,30 +3,29 @@ import { BEM } from '@redneckz/react-bem-helper';
...
       <div>There are no enabled plugins on this agent to collect the data from.</div>
-      <div>
-        To enable or install a plugin,&nbsp;
-        <AgentInfoLink to={`/agents/agent/${agent.id}/settings`}>
-          go back to Agent administration page.
-        </AgentInfoLink>
-      </div>
+      <div>To install a plugin go to</div>
     </Message>
+    <AgentInfoLink to={`/agents/${spacesToDashes(camelToSpaces(agentType))}/${agentId}/settings`}>
```
Comment (RomanAchilov) [inline src/modules/no-plugins-stub/no-plugins-stub.tsx:26]:
```suggestion
    <AgentInfoLink to={`/agents/${agentType === 'Agent' ? agent : service-group} /${agentId}/settings`}>
```

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — APPROVED** [review]

**Pull request merged**